static/app/views/explore/tables/tracesTable/fieldRenderers.tsx

```diff
@@ -71,8 +71,9 @@ export function ProjectsRenderer({
 }: ProjectsRendererProps) {
   const organization = useOrganization();
   const {projects} = useProjects({slugs: projectSlugs, orgId: organization.slug});
-  const projectAvatars =
```

Comment:
Worth adding a comment so someone in the future doesnt think its just inefficient :D